src/heading.js

```diff
@@ -40,23 +40,22 @@ export default class Heading extends Plugin {
 		const collection = new Collection();
 
 		// Add formats to collection.
-		for ( let format of formats ) {
+		for ( let { id, label } of formats ) {
 			collection.add( new Model( {
-				formatId: format.id,
-				label: format.label
+				formatId: id,
+				label: label
 			} ) );
 		}
 
 		// Create dropdown model.
 		const dropdownModel = new Model( {
-			label: 'Heading',
 			withText: true,
 			items: collection
 		} );
 
 		// Bind dropdown model to command.
 		dropdownModel.bind( 'isEnabled' ).to( command, 'isEnabled' );
-		dropdownModel.bind( 'label' ).to( command, 'value', format => format.label );
+		dropdownModel.bind( 'label' ).to( command, 'value', ( { label } ) => label );
```

Comment:
It's a matter of preference, I suppose. It's easier for me to read the second one.